sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/AadLoginTest.java

```diff
@@ -15,8 +16,9 @@
 
     @BeforeClass
     public static void setup() {
+        String endpoint = String.format("https://%s.blob.core.windows.net", ConfigurationManager.getConfiguration().get("PRIMARY_STORAGE_ACCOUNT_KEY"));
         storageClient = new BlobServiceClientBuilder()
-            .endpoint("https://" + System.getenv("ACCOUNT_NAME") + ".blob.core.windows.net")
+            .endpoint(endpoint)
             .credential(new EnvironmentCredential())
 //            .httpClient(HttpClient.createDefault().proxy(() -> new ProxyOptions(Type.HTTP, new InetSocketAddress("localhost", 8888))))
```

Comment:
Can we remove this line?

Reply:
This class is being deleted in another PR